iosApp/Survey/Sources/Presentation/Modules/SurveySelection/SurveyHeader/SurveyHeaderView.swift

```diff
@@ -14,18 +14,27 @@ struct SurveyHeaderView: View {
 
     let surveyHeader: SurveyHeaderUiModel
 
+    var imageAction: (() -> Void)?
+
     var body: some View {
         VStack(alignment: .leading, spacing: 4.0) {
             Text(surveyHeader.dateText)
                 .font(.boldSmall)
+                .accessibility(.surveySelection(.headerDateText))
             HStack {
                 Text(surveyHeader.todayText())
                     .font(.boldLarge)
+                    .accessibility(.surveySelection(.headerTitleText))
                 Spacer()
-                KFImage(surveyHeader.imageUrl.string.asURL)
-                    .resizable()
-                    .frame(width: 36.0, height: 36.0)
-                    .cornerRadius(18.0)
+                Button {
+                    imageAction?()
+                } label: {
+                    KFImage(surveyHeader.imageUrl.string.asURL)
```

Comment:
Optional: Create a new view or extension from Image to prevent importing third party in view.
```swift
extension Image {

    static func url(imagePath: String) -> some View {
		return KFImage(imagePath.asURL)
	}
}
```